alf/environments/alf_environment.py

```diff
@@ -218,6 +218,12 @@ def step(self, action):
         self._current_time_step = self._step(action)
         return self._current_time_step
 
+    def sync_progress(self):
+        """A placeholder interface to make it consistent with some derived classes
+        such as ``ParallelAlfEnvironment`` and ``FastParallelAlfEnvironment``.
+        """
+        pass
```

Comment:
suggest to raise error so that user knows something is wrong.

Reply:
> suggest to raise error so that user knows something is wrong.

Good point. Done.